pkg/kubelet/prober/worker.go

```diff
@@ -152,7 +152,7 @@ func (w *worker) doProbe() (keepGoing bool) {
 	}
 
 	c, ok := api.GetContainerStatus(status.ContainerStatuses, w.container.Name)
-	if !ok {
+	if !ok || len(c.ContainerID) == 0 {
 		// Either the container has not been created yet, or it was deleted.
 		glog.V(3).Infof("Non-existant container probed: %v - %v",
```

Comment:
I think unstarted is more accurate, since a waiting container exists in some sense, but doesn't have an ID since it hasn't started. A terminated container has started, but also stopped :)